en/docs/get-started/how-to-guides/ai-code-generation.md

```diff
@@ -0,0 +1,165 @@
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-code-gen/ai-code-gen-add-artifacts.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-code-gen/ai-code-gen-add-artifacts.png" alt="add artifacts" width="80%"></a>
+
+    You may refer to the following API, Email connection, and HTTP connection for reference.
+
+    ??? "WeatherEmailService API"
+        ```yaml
+        <api xmlns="http://ws.apache.org/ns/synapse" name="WeatherEmailService" context="/weatherEmail">
+            <resource methods="GET" uri-template="/getWeather?city={city}&amp;email={email}">
+                <inSequence>
+                    <variable name="API_KEY" type="STRING" value="REPLACE_WITH_YOUR_KEY"/>
+                    <http.get configKey="OpenWeather">
+                        <relativePath>/geo/1.0/direct?q=${params.queryParams.city}&amp;limit=1&amp;appid=${vars.API_KEY}</relativePath>
+                        <headers>[]</headers>
+                        <forceScAccepted>false</forceScAccepted>
+                        <disableChunking>false</disableChunking>
+                        <forceHttp10>false</forceHttp10>
+                        <noKeepAlive>false</noKeepAlive>
+                        <responseVariable>http_get_1</responseVariable>
+                        <overwriteBody>true</overwriteBody>
+                    </http.get>
+                    <http.get configKey="OpenWeather">
+                        <relativePath>/data/2.5/weather?lat=${payload[0].lat}&amp;lon=${payload[0].lon}&amp;appid=${vars.API_KEY}</relativePath>
+                        <headers>[]</headers>
+                        <forceScAccepted>false</forceScAccepted>
+                        <disableChunking>false</disableChunking>
+                        <forceHttp10>false</forceHttp10>
+                        <noKeepAlive>false</noKeepAlive>
+                        <responseVariable>http_get_2</responseVariable>
+                        <overwriteBody>true</overwriteBody>
+                    </http.get>
+                    <email.send configKey="EMAIL_CONN">
+                        <from>weather@example.com</from>
+                        <personalName></personalName>
+                        <to>{${params.queryParams.email}}</to>
+                        <cc></cc>
+                        <bcc></bcc>
+                        <replyTo></replyTo>
+                        <subject>Weather Update</subject>
+                        <content>{${payload}}</content>
+                        <contentType>text/html</contentType>
+                        <encoding>UTF-8</encoding>
+                        <attachments></attachments>
+                        <inlineImages>[]</inlineImages>
+                        <contentTransferEncoding>Base64</contentTransferEncoding>
+                    </email.send>
+                    <respond/>
+                </inSequence>
+            </resource>
+        </api>
+        ```
+
+    ??? "Email Connection"
```

Comment:
Suggestion: Since we are highly promoting low code development without giving xml syntax directly better to give UI form screenshot by blurring out sensitive information like username and passwords. Consider this for all local entries and HTTP endpoints.

Reply:
Fixed with b420ab6